noir-projects/noir-protocol-circuits/crates/types/src/abis/function_data.nr

```diff
@@ -7,12 +7,14 @@ use crate::{
 struct FunctionData {
     selector : FunctionSelector,
     is_private : bool,
+    is_static : bool,
```

Comment:
I think we don't need is_static in function data, but this can be removed later, it shouldn't be problematic (other that cost probably a few constraints for function tree hashing)